src/applications/accreditation/21a/sass/21a.scss

```diff
@@ -5,3 +5,7 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../../platform/forms/sass/m-form-confirmation";
+
+.yesNoDescription {
+  margin-bottom: -24px;
+}
```

Comment:
Minor point: can you fix the missing new line?

Reply:
Also, do we need this margin in the stylesheet, or can we move it to the component?